code/game/machinery/doors/airlock.dm

```diff
@@ -1289,7 +1289,7 @@ About the new airlock wires panel:
 	return ..()
 
 /obj/machinery/door/airlock/Uncross(atom/movable/mover)
-	if(density && ismob(mover) && !(mover.checkpass(PASSGLASS) && !opacity) && !(mover.checkpass(PASSDOOR)))
+	if(density && ismob(mover) && !(mover.checkpass(PASSGLASS) && !opacity) && !(mover.checkpass(PASSDOOR)) && !(istype(mover,/mob/living/simple_animal/shade)))//REEEEEEE
```

Comment:
pls no hardcoderino

Reply:
It's literally the only mob in the entire game with that property though. so either I had a snowflake exception here, or I had a snowflake flag or variable that'll only be used here as well. Which should it be?